### PR TITLE
Add keds progress bar

### PR DESCRIPTION
**Pull request by jmagalhaes-xing**

### CODE REVIEW

File: src/app/progress-bar/_progress-bar.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
+@import '../core/theming/theming';
+
+@mixin keds-progress-bar($theme) {
+  .keds-progress-bar {
+    background: keds-color-get($theme, core, shading, 20);
+    border-radius: 25px;
+    display: inline-block;
+    height: 4px;
+    position: relative;
```
Comment (RicardoNeves) [inline src/app/progress-bar/_progress-bar.scss]: why do we need this?

File: src/app/progress-bar/_progress-bar.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    width: 100%;
```
Comment (RicardoNeves) [inline src/app/progress-bar/_progress-bar.scss]: why?

Comment (jmagalhaes-xing) [reply src/app/progress-bar/_progress-bar.scss]: To fill the parent's size

**Comment by jmagalhaes-xing** [comment]
ready for testing

**Pull request merged**